moveit_ros/planning/trajectory_execution_manager/src/trajectory_execution_manager.cpp

```diff
@@ -137,7 +137,7 @@ void TrajectoryExecutionManager::initialize()
     if (controller == "moveit_ros_control_interface/MoveItControllerManager")
     {
       RCLCPP_WARN(LOGGER, "moveit_ros_control_interface/MoveItControllerManager is deprecated. Replace with "
-                          "`moveit_ros_control_interface/MoveItControllerManager.`");
+                          "`moveit_ros_control_interface/Ros2ControllerManager.`");
```

Comment:
This was my fault for suggesting the wrong thing! It should match what's here:

https://github.com/ros-planning/moveit2/pull/1601/files

```suggestion
                          "`moveit_ros_control_interface/Ros2ControlManager.`");
```

Reply:
i'll commit this now